src/els_config.erl

```diff
@@ -149,8 +149,12 @@ handle_cast(_Msg, State) -> {noreply, State}.
 %%==============================================================================
 
 -spec config_paths(path(), map()) -> [path()].
-config_paths(RootPath, #{<<"erlang">> := #{<<"config_path">> := ConfigPath}}) ->
-  [filename:join([RootPath, ConfigPath]) | default_config_paths(RootPath)];
+config_paths( RootPath
+            , #{<<"erlang">> := #{<<"config_path">> := ConfigPath0}}) ->
+  ConfigPath = binary_to_list(ConfigPath0),
+  lists:append([ possible_config_paths(ConfigPath)
+                , possible_config_paths(filename:join([RootPath, ConfigPath]))
```

Comment:
Nit: alignment of commas with `[`. 😄

Reply:
Need to get `erlfmt` integrated properly :)